src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -130,11 +130,12 @@ def pex_requirements(
         `all_requirements`.
         """
 
-        if not self.is_lockfile:
+        requirements = (*self.all_requirements, *extra_requirements)
+
+        if not self.uses_lockfile:
             return PexRequirements((*self.all_requirements, *extra_requirements))
```

Comment:
This still needs to use the new variable `requirements` to benefit from DRY

Reply:
FACEPALM